app/controllers/declaration/AddPersonController.scala

```diff
@@ -83,13 +106,15 @@ class AddPersonController @Inject () (
 
     dataCacheConnector.fetch[BusinessMatching](BusinessMatching.key) flatMap { bm =>
       val businessType = ControllerHelper.getBusinessType(bm)
+      val formWithModifiedErrors = form
```

Comment:
Why declare this? Unless the status method modifies it. In which case it is a `var`?